internal/pkg/model/component.go

```diff
@@ -28,6 +28,12 @@ type Component struct {
 	SchemaRow      json.RawMessage        `json:"configurationRowSchema,omitempty"`
 	EmptyConfig    *orderedmap.OrderedMap `json:"emptyConfiguration,omitempty"`
 	EmptyConfigRow *orderedmap.OrderedMap `json:"emptyConfigurationRow,omitempty"`
+	Data           ComponentData          `json:"data"`
+}
+
+type ComponentData struct {
+	DefaultBucket      bool   `json:"default_bucket"` //nolint
```

Comment:
tu by malo byt `//nolint: LINTER` aby bolo jasne, ktory vypinas. Asi nejaky JsonCamelCase ....

Reply:
Nazov zistis z vystupu `make lint`. Je na konci chybovej hlasky v `()`.

Reply:
Jo, jemu vadí ta podtržítka v jsonu (což moc nechápu když to vstupuje do aplikace zvenku 🙂 ). Ok, fixnu.

Reply:
Mozes ten linter cely vypnut, ked myslis ze nema zmysel.
Ale ten isty tag sa pouziva aj u suborov, ktore ukladame na disk, a tam sa to hodi.
Napr. `manifest.Content`.

Reply:
Go nevie co je schema inej aplikacie a co je nasa..

Reply:
Inak vidim, ze spadli testy na timeout `10m`, na interaktivnom teste ohladom `allowedBranches` dialogu s uzivatelom.
Pozriem sa na to, zatial je riesenie spustit to znova. Deje sa to nahodne.

Reply:
Aha, ok.

Reply:
Testy už prošly. 👍